### PR TITLE
Support on click deploy on Netlify #19

### PR DESCRIPTION
**Pull request by Riyan-Mo**

### CODE REVIEW

**Comment by ahaapple** [comment]
I think we should add the Required env variables name in the url. thanks

**Comment by Riyan-Mo** [comment]
I have added the variables in the url and also made a netlify.toml file where the environment variables can be obtained from.

File: frontend/netlify.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[template.environment]
+  UPSTASH_REDIS_REST_URL = ""
+  UPSTASH_REDIS_REST_TOKEN = ""
```
Comment (ahaapple) [inline frontend/netlify.toml:3]: please also add these envs, thanks.    the required envs are here:  https://github.com/memfreeme/memfree/blob/main/frontend/env-example

# https://serper.dev/ (Required)

# You could get the free SERPER_API_KEY in https://serper.dev/api-key

SERPER_API_KEY=

# Auth (Required)

# The AUTH_SECRET is a random string, you could use `openssl rand -base64 32` to generate it

AUTH_SECRET=

# OpenAi (Required) https://platform.openai.com

OPENAI_API_KEY=

**Comment by Riyan-Mo** [comment]
I added the rest of the variables. One thing I would like to ask is if I should include the netlify.toml file in gitignore?

**Review by ahaapple — APPROVED** [review]

**Comment by ahaapple** [comment]
@Riyan-Mo I think netlify.toml  could add to git, thanks.

**Pull request merged**